### PR TITLE
when columns removed for an incremental model

### PR DESCRIPTION
**Pull request by RatandeepSharma**
## What are you changing in this pull request and why?
If we remove a column from an incremental model and `on_schema_change: ignore` then execute dbt run, it results in an error. However, based on the documentation, it seems that dbt run should not fail in this scenario. I find this confusing.

##Link of the page
https://docs.getdbt.com/docs/build/incremental-models#default-behavior

### CODE REVIEW

**Comment by nghi-ly** [comment]
thanks for opening this PR, @RatandeepSharma ! appreciate your contribution to the dbt Labs docs! 

@joellabes : i think it's ok to approve and merge this change but want to double-check with you first. do you agree? or have any suggestions? thanks!

File: website/docs/docs/build/incremental-models.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ This is the behavior if `on_schema_change: ignore`, which is set by default, and
 
 If you add a column to your incremental model, and execute a `dbt run`, this column will _not_ appear in your target table.
 
-Similarly, if you remove a column from your incremental model, and execute a `dbt run`, this column will _not_ be removed from your target table.
+Similarly, if you remove a column from your incremental model, and execute a `dbt run`, `dbt run` will fail.
```
Comment (nghi-ly) [inline website/docs/docs/build/incremental-models.md]: suggestion: tiny cleanup for conciseness

```suggestion
If you remove a column from your incremental model and execute a `dbt run`, `dbt run` will fail.
```

File: website/docs/docs/build/incremental-models.md [hunk 2]
```diff
@@ -216,7 +216,7 @@ This is the behavior if `on_schema_change: ignore`, which is set by default, and
```
Comment (nghi-ly) [inline website/docs/docs/build/incremental-models.md:216]: Could you remove "and on older versions of dbt" in the sentence on line 215? This has been the default behavior for a while so is safe to remove. 

I was hoping to make it easier for you by using the built-in Add a suggestion but I can't select that line for some reason 🤷‍♀️

**Review by nghi-ly — CHANGES_REQUESTED** [review]
I chatted with joel today for a technical review. I've reviewed your PR and left some minor feedback. 

Thanks for catching this and letting us know, @RatandeepSharma ! 

Once you make the changes, we can ship it! ❇️

**Comment by mirnawong1** [comment]
Hey @RatandeepSharma wanted to check in to see if you can make those changes my teammate suggested?

File: website/docs/docs/build/incremental-models.md (hunk 2 above)
Comment (RatandeepSharma) [reply website/docs/docs/build/incremental-models.md:216]: Changes have been implemented as recommended.

**Comment by RatandeepSharma** [comment]
Hey @mirnawong1 - I've implemented the changes as recommended.

File: website/docs/docs/build/incremental-models.md [hunk 3]
```diff
@@ -212,11 +212,11 @@ Currently, `on_schema_change` only tracks top-level column changes. It does not
 
 ### Default behavior
 
-This is the behavior if `on_schema_change: ignore`, which is set by default, and on older versions of dbt.
+This is the behavior if `on_schema_change: ignore`, which is set by default.
```
Comment (mirnawong1) [inline website/docs/docs/build/incremental-models.md]:
```suggestion
This is the behavior of `on_schema_change: ignore`, which is set by default.
```

**Review by mirnawong1 — APPROVED** [review]
thanks again for making those changes and opening up this pr @RatandeepSharma ! great job and merging for you ✨

**Pull request merged**